src/classes/package_search.py

```diff
@@ -10,8 +10,8 @@
 
 # Default user credentials
 HOST = "localhost"
-USER = 'sdtAdmin'
-PASSWORD = "admin1234"
+USER = 'sdtreaduser'
+PASSWORD = "UPDATEME"
```

Comment:
lets make this to `CHANGEME` in sync with the instructions above

Reply:
I intentionally made them different so people would have to pay attention to the instructions instead of mindless copy/paste and ending up with an insecure installation :smile:

Reply:
agreed with @pleia2 , should I wait for @arshPratap to revert? I can revert it as well. Other than this, PR lgtm

Reply:
Is a revert needed? I think we can merge this as-is.

Reply:
oh yes my bad, I reversed the colours!